src/scenes/Event/store.js

```diff
@@ -369,16 +369,15 @@ export default class EventStore {
     }
 
     // Has not reached result setting start time
-    if ((status === ORACLE_RESULT_SETTING)
-      && currBlockTime.isBefore(moment.unix(resultSetStartTime))) {
+    if (status === PRE_RESULT_SETTING) {
       this.buttonDisabled = true;
       this.warningType = EventWarningType.INFO;
       this.eventWarningMessageId = 'oracle.setStartTimeDisabledText';
       return;
     }
 
     // User is not the result setter
-    if (status === ORACLE_RESULT_SETTING
+    if ([PRE_RESULT_SETTING, ORACLE_RESULT_SETTING].includes(status)
```

Comment:
Dont need to include `PRE_RESULT_SETTING` here. Should never hit here since the check on line 372.